UM/CentralFileStorage.py

```diff
@@ -129,10 +129,10 @@ def _getItemPath(cls, item_id: str, version: Version) -> str:
         :return: A path to store such an item.
         """
         item_name = item_id + "." + str(version)
-        return os.path.join(cls._centralStorageLocation(), item_name)
+        return os.path.join(cls.centralStorageLocation(), item_name)
 
     @classmethod
-    def _centralStorageLocation(cls) -> str:
+    def centralStorageLocation(cls) -> str:
```

Comment:
Nitpicking: Shouldn't it be getCentralStorageLocation? We tend to format properties like "centralStorageLocation" and functions as "getCentralStorageLocation"

Reply:
It's basically a read-only property. I didn't name it as such, I just removed the 'private to class' marker ;-)

Reply:
Fixed in https://github.com/Ultimaker/Uranium/pull/717/commits/81250f7af3c809cf213a6464016698bfeb9c0c12